data/map systems.txt

```diff
@@ -24476,6 +24477,39 @@ system Naper
 			distance 286
 			period 23.8403
 
+system Nashira
```

Comment:
There aren't really enough valuable asteroids in this system, I think. Maybe add some silver/gold?